src/server/ExposeAureliaDefinitions.ts

```diff
@@ -0,0 +1,256 @@
+  const { viewModel } = component;
+  if (!viewModel[propName]) return;
+
+  // 1.1 Find target path
+  const { scriptExtensions } = RelatedFileExtensions
+  const viewModelPath = component.paths.find(path => {
+    return scriptExtensions.some(ext => {
+      ext;
+      return path.endsWith(ext)
+    })
+  });
+  const targetUri = `file://${viewModelPath}`;
+
+  viewModel[propName].forEach(property => {
+    const { range } = property
+    const targetRange = Range.create(range.start, range.end);
+
+    const def = definitionsInfo[property.name] || [];
+    def.push(
+      LocationLink.create(
+        targetUri,
+        targetRange,
+        targetRange,
+      )
+    );
+
+    definitionsInfo[property.name] = def;
+  });
+  return definitionsInfo;
+}
+
+/**
+ * Store definitions from an aurelia view, eg.
+ * - 1. `repeat.for=">>>button<<< of buttons"`
+ * - 2. `<div >>>my-attr<<<.bind=""></div>
+ */
+function storeViewDefinitions(
+  component: WebComponent,
+  aureliaApplication: AureliaApplication,
+  definitionsInfo: DefinitionsInfo,
+  definitionsAttributesInfo: DefinitionsAttributesInfo,
+) {
+  const viewDocument = component.document;
+  if (!viewDocument) return;
+
+  const { scriptExtensions } = RelatedFileExtensions;
+  const tags = viewDocument.tags.filter(tag => {
+    const isDontParseTags = (tag.name === 'template') || (tag.name === 'require');
+    const isStartTag = !isDontParseTags && tag.startTag;
+    return (!isDontParseTags && tag.startTag) // omit closing tags
+  });
+
+  tags.forEach(tag => {
+    const customElementName = tag.name;
+    tag.attributes.forEach(attr => {
+      /**
+       * 1.
+       * TODO: Cursor at the end of the line, and not directly where the value was defined, ie.
+       * `repeat.for="button of buttons"|`
+       * But we want `repeat.for="|button of buttons"`
+       * Fix approach: get startOffSet from `repeat.for` attribute
+       * (because parse5 only gives us attr positions)
+       * then + 12 (= num of char in `repeat.for="`) to get the beginning of value.
+       */
+      if (attr.name === 'repeat') {
+        /* eg. value = 'button of buttons' */
+        const { value } = attr;
+        const definitionWord = value.split(' ')[0]; /* button */
+        const targetViewUri = `file://${viewDocument.path}`;
+        /* parse5 line index starts 1 */
+        const lineNum = tag.line - 1;
+
+        const targetRange = Range.create(
+          Position.create(lineNum, attr.startOffset),
+          Position.create(lineNum, attr.endOffset),
+        );
+
+        const def = definitionsInfo[definitionWord] || [];
+        def.push(
+          LocationLink.create(
+            targetViewUri,
+            targetRange,
+            targetRange,
+          )
+        );
+        definitionsInfo[definitionWord] = def;
+      }
+
+      /*
+       * 2.
+       * Here, I assume, that .binding will only be true for valid attr. bindings
+       */
+      if (attr.binding === 'bind') {

```

Comment:
> 4. limited to .bind
Should include .call, .two-way, ...

Should add an array with all possible bindings here